### PR TITLE
make getToken public to simplify interacting with the API

### PR DESCRIPTION
**Pull request by anarcat**
this is necessary to POST new entries for example

### CODE REVIEW

File: token.go [hunk 1]
```diff
@@ -20,6 +20,8 @@ type Token struct {
 	RefreshToken string `json:"refresh_token"`
 }
 
+// GetToken will use the credentials set in the configuration to
+// request an acces token from the Wallabag API
```
Comment (Strubbl) [inline token.go]: please change: 
* acces --> access
* Wallabag --> wallabag

**Review by Strubbl — COMMENTED** [review]
Sorry, i just saw your PR now. I was working in parallel. Can you please rebase?

**Comment by anarcat** [comment]
okay - i completely rewrote this since you introduced a much more useful function that I can use directly. in the end, maybe all this will be made useless by #5, but for now it would be nice to see this merged anyways...

**Comment by Strubbl** [comment]
OKay, let's merge that for now. Later on a refactoring to your suggested doAPI() (in #5) will be done. Thanks!

**Pull request merged**